thrall/app/lib/ThrallConfig.scala

```diff
@@ -28,6 +28,11 @@ class ThrallConfig(override val configuration: Configuration) extends CommonConf
         "App"   -> Seq(elasticsearchApp)
       ))
 
+
+  lazy val elasticsearch6Host: String = properties.getOrElse("es6.host", "localhost")     // TODO how to discover the 6 cluster in AWS
```

Comment:
We'll have a load balancer in front of the ES cluster so could do a describe by tag?